src/main/scala/codacy/codenarc/CodeNarcOutput.scala

```diff
@@ -15,7 +16,7 @@ object CodeNarcOutput {
         case violation: Violation =>
           CodeNarcOutput(
             fileResults.getPath,
-            violation.getMessage,
+            Option(violation.getMessage).getOrElse("Not available"),
```

Comment:
`"Not available"` ?
Doesn't make more sense an empty string ?
Also, do we know if Codacy allows to undefined `message`s for Patterns ?

Reply:
I would really prefer a "N/A" or "not available" then an empty string. Actually I prefer just "N/A".

I am a bit concern with an empty string making something fail as well, and then we assume that this didn't fix anything...